### PR TITLE
add LICENSE to system1 specific added code for sp-privacy

### PR DESCRIPTION
**Pull request by danielrussellLA**

### CODE REVIEW

File: sp-privacy/LICENSE [hunk 1]
```diff
@@ -0,0 +1,845 @@
...
+
+Except for the license granted herein to System1 and recipients of software
+distributed by System1, You reserve all right, title, and interest in and to Your
+Contributions.
+
+If you do not agree to these terms, please do not send patches or pull
+requests, or commit any material to this repository.
+
+LICENSES FOR INCORPORATED CODEBASES
+===================================
```
Comment (drmonkeysee) [inline sp-privacy/LICENSE:30]: i wouldn't include all of these unless we actually pulled them into the privacy badger repo, only reference any 3rd parties we've added to this codebase

File: sp-privacy/LICENSE [hunk 2]
```diff
@@ -0,0 +1,845 @@
+sp-privacy
+Copyright © 2021 System1 and other contributors
```
Comment (drmonkeysee) [inline sp-privacy/LICENSE:2]: this might need to be Surfboard Holdings BV instead?

Comment (danielrussellLA) [reply sp-privacy/LICENSE:2]: ah good point, i'll update

**Comment by drmonkeysee** [comment]
We should also add the copyright notice to all our source-code files in sp-privacy:

```/*
 * Copyright (C) 2022 Surfboard Holding B.V. <https://www.startpage.com>
 *
 * This program is free software: you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation, either version 3 of the License, or
 * (at your option) any later version.
 *
 * This program is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 *
 * You should have received a copy of the GNU General Public License
 * along with this program.  If not, see <https://www.gnu.org/licenses/>.
 */
```

this file https://github.com/Openmail/privacybadger/blob/master/sp-privacy/sp-js/socialwidgetsOverrides.js currently has the privacy badger copyright notice in it but i think that's wrong, this should be our copyright notice since it's our file

**Review by drmonkeysee — APPROVED** [review]

**Pull request merged**